README.md

```diff
@@ -39,6 +39,11 @@ export const routes: Routes = [
 ];
 ```
 
+**NOTE**: When specifying a relative path to lazy loaded module, one of the following two conditions *must* hold:
+
+* The lazy loaded route is defined in the same module file where it is added by `RouterModule.forXxx()`
```

Comment:
Not sure what you are trying to say here. You don't need to do anything special if you're including the routes in the module file.

If you are putting your routes in a separate file, then that file should be a sibling of the module file where it will be imported.